content/docs/user-guide/tips-and-tricks.md

```diff
@@ -0,0 +1,17 @@
+# Tips and tricks for DVC Projects
+
+This guide provides general tips and tricks related to DVC, which can be
+utilized while working on a project. Using the practices listed here, you can
+manage your projects with DVC more efficiently.
+
+## Using meta in dvc.yaml or .dvc files
+
+DVC provides an optional `meta` field in `dvc.yaml` and `.dvc` file. It can be
+used to add any user specific information. It also supports YAML content.
+
+## Switching between datasets
+
+You can quickly switch between a large dataset and a small subset without
+modifying source code. To achieve this yoe need to change dependencies of
```

Comment:
yoe -> please install a spell checked in your IDE @imhardikj 🙂

Reply:
Thanks. Strangely the spell checker didn't flag this (might have mistaken it for abbreviation).

Reply:
I see this is fixed in https://github.com/iterative/dvc.org/pull/1748/files#diff-c1d825a488a38eed05f19d3d2fbe8b42